ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -214,15 +185,15 @@ export function AssetPicker({
             <Text className="asset-picker__symbol" variant={TextVariant.bodyMd}>
               {formattedSymbol}
             </Text>
-            {asset.details?.tokenId && (
+            {isNFT && asset?.tokenId && (
```

Comment:
Only tokens that have tokenIds are NFTS if I'm not mistaken. 
Can we rely on `isNFT`?

Reply:
That's right - tokenId is only in NFTs. Just needed to add the tokenId check here to avoid a type error on line 194